src/microbe_stage/ProcessList.cs

```diff
@@ -103,6 +125,22 @@ private ChemicalEquation CreateEquation(StrictProcessDisplayInfoEquality process
         return equation;
     }
 
+    private void UpdateEquationsExternalSpeedModifier()
+    {
+        if (createdProcessControls == null!)
+            return;
+
+        if (ExternalSpeedModifier == previousExternalSpeedModifier)
```

Comment:
This is unnecessary, the check already exists in the setter that avoids duplicate values being set and calling UpdateEquationsExternalSpeedModifier again.

Reply:
I'll change this myself so I can do a final test on this PR (as I need to soon wrap up work for today).